auto-merge.example.yml

```diff
@@ -46,3 +46,6 @@ blockingLabels:
 # all of these labels are attached to a pull request.
 requiredLabels:
 - merge
+
+# Automatic merges will be blocked if there is a match between the regular expression and title
+blockingTitleRegex: 'wip'
```

Comment:
It would be nice to make `wip` match a word for the default example. This would mean using something like:

```
blockingTitleRegex: '\bwip\b'
```

That would match `wip my pr`, `[wip] my pr`, but not `add swiping`.

Also, could you add a line at the end of file?